lib/ocamlorg_web/graphql.mli

```diff
@@ -11,23 +11,27 @@ type packages_result =
   }
 
 val starts_with : string -> string -> bool
-(** This function compares two strings and returns true if the first string is
-    equal to the beginning of the second string up to capitalization *)
-
-val is_package : string -> string -> bool
 (** This function compares two strings and returns true if they are equal up to
     capitalization *)
 
+val is_package : string -> string -> bool
+(** This function compares two strings and returns true if the first string is
+    equal to the beginning of the second string up to capitalization *)
+
```

Comment:
I think the comments should stay the other way around. You fixed that in the other PR and here you seem to go back to having the comment of `start_with` for `is_package` and vice-versa again. Maybe a mix-up during the rebase?

Reply:
Didn't see that, sorry.
It's fixed, thank you